packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -469,9 +471,12 @@ export default {
                     address: this.address,
                     customer: this.customer,
                     isCheckoutMethodDelivery: this.isCheckoutMethodDelivery,
-                    time: this.time,
+                    time: this.hasAsapSelected
+                        ? this.availableFulfilment.times[0]
+                        : this.time,
```

Comment:
Not sure I understand this. Aren't we always sending `this.time`, and the only thing that should change is whether we send `asap: true` or `asap: false`?

Reply:
If asap is set to true then we need to send the time, at the moment without this the time isn't sent. This will send the first time available in the `availableFulfilment` array.

Reply:
Yeah, that's what I mean, we need to send the `from` and `to` in every case. I'd expect these to be in the store and sent all the time. Happy to take it offline.

Reply:
Yup understand that part but that's the issue, the `from` & `to` aren't set in the store until the dropdown changes. So if we want that to always be set the `selection.vue` component needs updating so instead of watching for changes we either set the first time as a new value and read from that (similar to `this.availableFulfilment.times[0]`) or we set it this way as it's already been set in the store via `availableFulfilment` which contains the whole list of times (we only need the one).

Here's what is returns if we revert this change:

![Uploading Screen Shot 2021-03-24 at 3.22.04 PM.png…]()

Reply:
I see. As you said in your comment, I think the `from` and `to` should always be set, and that would make the code simpler. What do you think?

Reply:
Agree. I'll update selection component so it updates correctly.

Reply:
Fixed.